ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -571,4 +571,14 @@ FAN_SS_AL:
   - control_mode
   - run_mode
   - schedule_run_command
-  - control_status
\ No newline at end of file
+  - control_status
+
+FAN_SSSCFASR:
+  description: "Basic Exhaust fan with start/stop, speed command, fail alarm, scheduled run"
+  uses:
+  - exhaust_fan_speed_percentage_command
```

Comment:
Shouldn't this be `FAN_SS_SRC`? And, if so, wouldn't it match the type directly above it (`FAN_SS_AL`)?

Reply:
Hi sir, sorry for late reply. I have an extra point needed that differs from AL:
```
uses:
  - exhaust_fan_speed_percentage_command
```

I also don't need some in the opt_uses, so I decided to make a new entry, i think it should be like this now:
```
FAN_SS_SPS_FA_SR:
  description: "Basic Exhaust fan with start/stop, speed sensor, fail alarm, scheduled run"
  uses:
  - exhaust_fan_speed_percentage_sensor
  - lost_power_alarm
  implements:
  - FAN
  - SS
  - SRC
```
Is it better to create a new type  for my specific use like above?
Or is it better to re-use exisiting types like FAN_AL (has some points more than is needed) and just apply MISSING on some points later, like below maybe?
```
FAN_SS_AL_SPS_SR:
  description: "Basic Exhaust fan with start/stop, fail alarm, speed sensor, scheduled run"
  uses:
  - exhaust_fan_speed_percentage_sensor
  implements:
  - FAN_SS_AL
  - SRC
```